tests/test_controlnet_inferers.py

```diff
@@ -975,24 +977,27 @@ def test_prediction_shape_conditioned_concat(
         autoencoder_params,
         dm_model_type,
         stage_2_params,
-        controlnet_params,
+        cn_params,
```

Comment:
what was the motivation behind renaming controlnet_params to cn_params? if its to become consistent with usage elsewhere that seems fair, but if not, i think it just makes it a bit less readable